app/models/auction.rb

```diff
@@ -0,0 +1,63 @@
+class Auction < ActiveRecord::Base
+  enum status: {
+    started: 'started',
+    awaiting_payment: 'awaiting_payment',
+    no_bids: 'no_bids',
+    payment_received: 'payment_received',
+    payment_not_received: 'payment_not_received',
+    domain_registered: 'domain_registered',
```

Comment:
We need additional state, `domain_not_registered`, which would functionally do the same thing as `payment_not_received`, create a new auction for the same domain name. @vohmar 

There's instrumentation already for that on auction center's side.